### PR TITLE
List invoice checkbox variant

### PR DESCRIPTION
**Pull request by TChukwuleta**
Fix for PR #3473 (Make Payment Request list view consistent with Invoices by adding mass action (Archive, Clone?))

### CODE REVIEW

File: BTCPayServer/Views/UIInvoice/ListInvoices.cshtml [hunk 1]
```diff
@@ -37,11 +36,40 @@
             padding-left: 1rem;
             text-align: left;
         }
+        .dropdown-item.bold {
+            font-weight: bold;
+        }
+        .dropdown-item + .dropdown-item {
+            margin-top: 5px;
+        }
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIInvoice/ListInvoices.cshtml:44]: Please do not add things like that. We should stick to what the customized version of Bootstrap provides to stay consistent. If we decide those changes should be general, then we'd need to add them to the [design system](https://design.btcpayserver.org).

Comment (TChukwuleta) [reply BTCPayServer/Views/UIInvoice/ListInvoices.cshtml:44]: Noted. Would make the update

**Comment by Serheij** [comment]
Немогу вывести деньги в чем проблема уже месяц мучаюсь и не могу ничего.

**Comment by Serheij** [comment]
Как вывести деньги

File: BTCPayServer/Views/UIInvoice/ListInvoices.cshtml [hunk 2]
```diff
@@ -42,6 +42,29 @@
...
+            align-items: center;
+        }
+        .functionality-icon {
+            margin-right: 5px;
+        }
+        .functionality-text {
+            font-weight: bold;
+            margin-right: 5px;
+            white-space: nowrap;
+        }
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIInvoice/ListInvoices.cshtml:67]: Please cover these cases by applying our Bootstrap classes to the elements.

Comment (TChukwuleta) [reply BTCPayServer/Views/UIInvoice/ListInvoices.cshtml:67]: Sure I would.

**Review by dstrukt — APPROVED** [review]
Haven't done an audit of the code, but visually and functionally this seems to work great!

**Review by dstrukt — COMMENTED** [review]
Appears https://github.com/btcpayserver/btcpayserver/pull/5399 will supersede, this should not be merged.

**Pull request closed without merging**